pkg/qemu/qemu.go

```diff
@@ -206,16 +206,20 @@ func Cmdline(cfg Config) (string, []string, error) {
 		}
 		return "", nil, errors.New(errStr)
 	}
+
+	var cpu string
```

Comment:
I think the whole default handling to determine the definitive cpu type should happen in `pkg/limayaml/defaults.go`, including the normal `default.yaml` and `override.yaml` rules. The final value should be reported by `limactl ls --json`.

Reply:
Thanks for the suggestion, I will improve it.